### PR TITLE
[prismjs] Allow to export static writable properties

### PR DESCRIPTION
**Pull request by martin-badin**
Discussion: #58405
Documentation: 
https://prismjs.com/docs/Prism.html#.disableWorkerMessageHandler
https://prismjs.com/docs/Prism.html#.manual
https://prismjs.com/#basic-usage-bundlers

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test prismjs`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

### CODE REVIEW

File: types/prismjs/index.d.ts [hunk 1]
```diff
@@ -1,385 +1,395 @@
...
-
-    /**
-     * If `true`, then the first capturing group of `pattern` will (effectively) behave as a lookbehind
-     * group meaning that the captured text will not be part of the matched text of the new token.
-     */
-    lookbehind?: boolean | undefined;
+declare namespace Prism {
+    const languages: Languages;
+    const plugins: Record<string, any>;
+    let disableWorkerMessageHandler: boolean;
```
Comment (srmagura) [inline types/prismjs/index.d.ts:13]: Both `disableWorkerMessageHandler` and `manual` start out as undefined, so the type should be `boolean | undefined`.

**Review by srmagura — CHANGES_REQUESTED** [review]
A bit hard to follow the diff, but looks good. Will close #58417 when this gets merged.

File: types/prismjs/prismjs-tests.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import * as Prism from 'prismjs';
```
Comment (andrewiggins) [inline types/prismjs/prismjs-tests.ts:1]: Is this pattern still supported or is this a breaking change?

Comment (martin-badin) [reply types/prismjs/prismjs-tests.ts:1]: Hi. The pattern isn't supported but I need to find a different way how to do that because this pattern isn't supported too `const Prism = require('prismjs');`

File: types/prismjs/index.d.ts (hunk 1 above)
Comment (martin-badin) [reply types/prismjs/index.d.ts]: Are you sure? The default value for both variables is `false`.
https://github.com/PrismJS/prism/blob/master/prism.js#L51-L78

File: types/prismjs/prismjs-tests.ts (hunk 2 above)
Comment (martin-badin) [reply types/prismjs/prismjs-tests.ts:1]: Now are supported both patterns
`import * as Prism from 'prismjs';`
`import Prism = require('prismjs');`
but the new variables are writable only if I use `import Prism = require('prismjs');`

File: types/prismjs/index.d.ts (hunk 1 above)
Comment (srmagura) [reply types/prismjs/index.d.ts]: image

File: types/prismjs/index.d.ts [hunk 3]
```diff
@@ -148,6 +175,41 @@ export namespace util {
 
     /** Deep clone a language definition (e.g. to extend it) */
     function clone<T>(o: T): T;
+
+    /**
+     * Returns the Prism language of the given element set by a `language-xxxx` or `lang-xxxx` class.
+     *
+     * If no language is set for the element or the element is `null` or `undefined`, `none` will be returned.
+     */
+    function getLanguage(element: Element): string;
```
Comment (RunDevelopment) [inline types/prismjs/index.d.ts:184]: No. These are internal utility functions that might change at any time. They are only exported here because some plugins need these functions.

They are not part of the public API and should not be used by others.

The other util function are technically internal as well, but they are quite old and have existed since before we drew a clear line between public and private API. They are documented here not because people should use them, but because we don't want to break the code of people that might rely on it.

Comment (martin-badin) [reply types/prismjs/index.d.ts]: thanks. I will remove that

**Review by RunDevelopment — APPROVED** [review]

**Comment by martin-badin** [comment]
Ready to merge
